cmd/launcher/main.go

```diff
@@ -97,9 +99,30 @@ func main() {
 	// recreate the logger with  the appropriate level.
 	logger = logutil.NewServerLogger(opts.Debug)
 
+	// set up system slogger to write to os logs
+	systemSlogger := new(multislogger.MultiSlogger)
+	systemSlogger.AddHandler(slog.NewJSONHandler(os.Stderr, &slog.HandlerOptions{
+		Level: slog.LevelInfo,
+	}))
+
+	// set up local slogger to write to debug.json
+	localSlogger := new(multislogger.MultiSlogger)
```

Comment:
```suggestion
	// set up slogger for internal launcher logging
	slogger := new(multislogger.MultiSlogger)
```

This should be the one people reach for first

Reply:
done